src/test/java/org/springframework/samples/petclinic/rest/controller/OwnerRestControllerTests.java

```diff
@@ -339,8 +338,8 @@ void testDeleteOwnerError() throws Exception {
         ObjectMapper mapper = new ObjectMapper();
         mapper.registerModule(new JavaTimeModule());
         String newOwnerAsJSON = mapper.writeValueAsString(newOwnerDto);
-        given(this.clinicService.findOwnerById(-1)).willReturn(null);
-        this.mockMvc.perform(delete("/api/owners/-1")
+        given(this.clinicService.findOwnerById(999)).willReturn(null);
```

Comment:
Why do you change the `-1` value by `99`?

Reply:
I did that because of the restriction added by the generated code

```java
default ResponseEntity<PetTypeDto> getPetType(@Min(0)@ApiParam(value = "The ID of the pet type.",required=true) @PathVariable("petTypeId") Integer petTypeId) {
```